bitnami/fluentd/Chart.yaml

```
@@ -30,4 +30,4 @@ maintainers:
 name: fluentd
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/fluentd
-version: 6.1.1
+version: 6.2.1
```

Correct version bump:
```suggestion
version: 6.2.0
```